cvat/apps/auto_annotation/model_manager.py

```diff
@@ -44,11 +46,12 @@ def _delete_source_files():
     def _run_test(model_file, weights_file, labelmap_file, interpretation_file):
         test_image = np.ones((1024, 1980, 3), np.uint8) * 255
         try:
+            dummy_label_map = {key: key for key in load_label_map(labelmap_file).keys()}
```

Comment:
Could you please use the same naming convention? For example, labelmap_file, labelmap, dummy_labelmap. Why dummy_label_map? Is it a real one?